### DISCUSSION TITLE
Anime4K in the browser

### DISCUSSION THREAD

**Discussion by sb2702** [bloc97/Anime4K#223] [Show and tell]
I put together a free tool which should upscale with the [super ultra light version of the CNN based upscaler](https://github.com/bloc97/Anime4K/blob/master/glsl/Upscale/Anime4K_Upscale_CNN_x2_S.glsl) and output the resulting upscaled file

https://free.upscaler.video/

<img width="600" alt="kurzgesagt" src="https://github.com/bloc97/Anime4K/assets/5678502/24517039-0cff-4550-8d0a-71c0128649b0">

Source code is [here](https://github.com/sb2702/free-ai-video-upscaler), based on my [WebGPU SDK](https://github.com/sb2702/websr) / implementation of that network.

More networks coming soon, but I wanted to get something minimally useful out there to start improving.

I'm also thinking about doing a Chrome extension to add real-time upscaling to videos on YouTube, Vimeo etc...

**Comment by Tama47** [comment]
Nice tool, hope to see more network soon! 

But I think there's some audio issues with the upscaled video?

> I'm also thinking about doing a Chrome extension to add real-time upscaling to videos on YouTube, Vimeo etc...

Let me know the progress on this, really excited for it!
